cmd/languageserver.go

```diff
@@ -25,6 +27,23 @@ func init() {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
+			exe, err := os.Executable()
+			if err != nil {
+				fmt.Fprintln(os.Stderr, "error getting executable:", err)
+			} else {
+				absPath, err := filepath.Abs(exe)
+				if err != nil {
+					fmt.Fprintln(os.Stderr, "error getting executable path:", err)
+				} else {
+					v := version.Version
+					if v == "" {
+						v = "Unknown"
+					}
```

Comment:
Do you know [`cmp.Or`](https://pkg.go.dev/cmp#Or)? We could write this as
```suggestion
					v := cmp.Or(version.Version, "Unknown")
```

Reply:
Oh, nice! Had no idea that was a stdlib thing. I had already merged this but we could easily sprinkle that in lots of places I'm sure.